nav_msgs/Path poses
std_msgs/String new_frame
---
nav_msgs/Path poses
